.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "bitwise": true,
```

Comment:
I'd rather use eslint if we're going to add a linter. Eslint is currently being used on `starwipe-cms`:

- https://github.com/theonion/starwipe-cms/blob/master/scripts/lint
- https://github.com/theonion/starwipe-cms/blob/master/resources/js/eslint.json
- https://github.com/theonion/starwipe-cms/blob/master/resources/js/eslint-dev.json

`scripts/lint` is docker specific, but with eslint as a node dep, you should be able to run the same commands.

There's definitely a way we could unify the rules across all sites, but until then, we should try to use the same configs.